mods/ts/rules/nod-vehicles.yaml

```diff
@@ -118,6 +119,7 @@ ART2:
 		Cost: 975
 	Tooltip:
 		Name: Artillery
+		Description: Mobile Artillery. Needs\nto deploy in order to shoot.\n  Strong vs ground units\n  Cannot target Aircraft
```

Comment:
Will this fit in the tooltip if you moved "Needs" to the second line, so it's a sentence per line?